### ISSUE TITLE
Chrome user console error - think this is just speed.

### ISSUE THREAD

**Issue by tomstarley** [SmoofCreative/kasban#253]
![screen shot 2016-05-25 at 08 42 56](https://cloud.githubusercontent.com/assets/12812427/15531880/b5a96c6c-2254-11e6-88cc-4396d6eeb0c9.png)

**Comment by tomstarley** [comment]
We had another person who cant load a couple of projects. Finding out the details.

**Comment by tomstarley** [comment]
Just had another.. see ppmessage. Seems he cant load a couple of projects, some of them are not big - 16 tasks... But if he logs out of his account and into the main assana account they load fine.

**Comment by tomstarley** [comment]
One more for this! @alexdunne i think we need to make this one our priority. Can you look at this tonight?

**Comment by alexdunne** [comment]
We could do with getting an idea of the structure of these projects that are breaking. Is the user with 16 projects actively responding? If so we could do with do with replicating his project to see if we can replicate the issue.

**Comment by tomstarley** [comment]
Yup he is! Theres a couple, can you take a look at ppm and catch up with them?
